#Request
---
#Respose
bool exploration_done
geometry_msgs/PoseArray bestViewPose
